lib/home/home_page.dart

```diff
@@ -23,38 +25,100 @@ class HomePage extends StatelessWidget {
   }
 }
 
-class HomeView extends StatelessWidget {
+class HomeView extends StatefulWidget {
   const HomeView({super.key});
 
+  @override
+  State<HomeView> createState() => _HomeViewState();
+}
+
+class _HomeViewState extends State<HomeView> {
+  late ScrollController _scrollController;
+  double expandedHeight = 132;
+  bool lastStatus = true;
```

Comment:
What is this variable about? Sounds unused

Reply:
Oh it is used to call setState only if necessary.

In that case, this should be the `isShrunk` variable, we don't need the getter 😎